django_prices_vatlayer/migrations/0002_auto_20170120_1019.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-20 10:19
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_prices_vatlayer', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name='Vat',
+            new_name='EuropeanVatRates',
+        ),
+    ]
```

Comment:
After renaming models, mgirations should be regenerated to have only one migration that introduces a model with correct name.